### PR TITLE
Repository moved to different url

### PR DESCRIPTION
**Pull request by datenbrille**
I hope I did the right thing with the versions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,15 +11,15 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>startbootstrap-sb-admin-2</artifactId>
-    <version>1.0.8-SNAPSHOT</version>
+    <version>1.0.9-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `1.0.8-SNAPSHOT` since Maven does the actual release.

Comment (datenbrille) [inline pom.xml]: Reverted the change

File: pom.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <upstream.version>1.0.7</upstream.version>
```
Comment (jamesward) [inline pom.xml]: You'll need to bump this to `1.0.8` if you want a release for `1.0.8`.

Comment (datenbrille) [inline pom.xml]: Thanks, is there a tutorial how webjars work and how to improve them?

**Pull request merged**